### ISSUE TITLE
Usage Example

### ISSUE THREAD

**Issue by erinyoung** [boasvdp/assembly_snptyper#1]
I am unsure if I'm using this tool correct.

I have ~250 fastas for GAS samples. I listed them one per line in the fastas.txt file.

I then ran the following 
```
assembly_snptyper --list_input fastas.txt -p 20 --reference ~/assembly_snptyper/data/MGAS5005.fa.gz --vcf ~/assembly_snptyper/data/M1UK.vcf > m1uk.txt
```

But I got 0 matching SNPs, suggesting that I'm using this tool wrong.

Can you correct me?

**Comment by boasvdp** [comment]
Hi Erin, thanks for checking out this tool!

It seems the script didn't pick up on a gzipped ref and just reported zero matches without raising an error. I've included a check before running minimap2 to see whether the ref is a flat text file. This will now raise an error to indicate the ref should be unzipped if it can't be read by the Python code prior to running minimap2.

I've also unzipped the MGAS5005 ref included in the repo to prevent confusion, and added this to the README. Thanks for reporting this!
